projects/rero/ng-core/src/lib/record/editor/editor.component.ts

```diff
@@ -341,8 +341,8 @@ export class EditorComponent implements OnInit, OnChanges, OnDestroy {
     if (
       !['object', 'multischema', 'array'].some(f => f === field.type)
     ) {
-      // TODO following line seems to raise an error with v5.9.0 of ngx-formly
-      model = field.model[field.key];
+      // New from ngx-formly v5.9.0
+      model = field.model[Array.isArray(field.key) ? field.key[0] : field.key];
```

Comment:
Could you please explain this change in commit message?

Reply:
Done